src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -1258,10 +1258,10 @@
   "NotificationsEmailSettingsName": "Email",
   "NotificationsEmailSettingsRecipientAddress": "Recipient Address(es)",
   "NotificationsEmailSettingsRecipientAddressHelpText": "Comma separated list of email recipients",
-  "NotificationsEmailSettingsRequireEncryption": "Require Encryption",
-  "NotificationsEmailSettingsRequireEncryptionHelpText": "Require SSL (Port 465 only) or StartTLS (any other port)",
   "NotificationsEmailSettingsServer": "Server",
   "NotificationsEmailSettingsServerHelpText": "Hostname or IP of Email server",
+  "NotificationsEmailSettingsUseEncryption": "Use Encryption",
+  "NotificationsEmailSettingsUseEncryptionHelpText": "Whether to optimistically use encryption if configured on the server, to always use encryption via SSL (Port 465 only) or StartTLS (any other port) or to never use encryption",
```

Comment:
```suggestion
  "NotificationsEmailSettingsUseEncryptionHelpText": "Whether to Optimistically (auto) use encryption if configured on the server; to Always use encryption via SSL (Port 465 only) or StartTLS (any other port); or to Never use encryption",
```

UX clarity

Reply:
Auto is not 100% correct for optimistically, since it may be confused with falling back to insecure connections if the secure connections fail. We're discussing alternatives instead of `Optimistically`.